addon/components/bread-crumbs.js

```diff
@@ -79,7 +81,22 @@ export default Component.extend({
   },
 
   _lookupRoute(routeName) {
```

Comment:
```
/*
 * Lookup local route first and fallback to engine,
 * I'm not exactly familiar with local vs engine routes,
 * but my thinking is you should be able to override an
 * engine route locally so it should take priority.
 * I could be totally wrong here...
 */
_lookupRoute(routeName) {
  return _lookupLocalRoute(routeName) || _lookupEngineRoute(routeName);
},

_lookupLocalRoute(routeName) {
  return getOwner(this).lookup(`route:${routeName}`);
},

_lookupEngineRoute(routeName) {
  const router = get(this, 'routing.router');

  let engineInfo = router._engineInfoByRoute[routeName];

  if (!engineInfo) {
    return;
  }

  return router
    ._getEngineInstance(engineInfo)
    .lookup(`route:${engineInfo.localFullName);
}
```

Reply:
@dguettler Aside from the conflict, is this what's holding up the PR?

Just trying to understand whether the issue is trying to determine the right lookup and fallback approach?